suites/quincy/nvmeof/ceph_nvmeof_deployment.yaml

```diff
@@ -95,6 +97,45 @@ tests:
       name: configure Ceph NVMEoF Gateway
       polarion-id:
 
+# Run IOs using multiple-initiators against multi-subsystems based NVMe-OF targets using fio tool
+  - test:
+      abort-on-fail: true
+      config:
+        gw_node: node6
+        rbd_pool: rbd
+        do_not_create_image: true
+        rep-pool-only: true
+        rep_pool_config:
+          pool: rbd
+        install: true                           # Run SPDK with all pre-requisites
+        subsystems:                             # Configure subsystems with all sub-entities
+          - nqn: nqn.2016-06.io.spdk:cnode2
+            serial: 2
+            bdevs:
+              count: 1
+              size: 10G
+            listener_port: 5002
+            allow_host: "*"
+          - nqn: nqn.2016-06.io.spdk:cnode3
+            serial: 3
+            bdevs:
+              count: 1
+              size: 10G
+            listener_port: 5003
+            allow_host: "*"
+        initiators:                             # Configure Initiators with all pre-req
+          - subnqn: nqn.2016-06.io.spdk:cnode2
+            listener_port: 5002
+            node: node8
+          - subnqn: nqn.2016-06.io.spdk:cnode3
+            listener_port: 5003
+            node: node9
+      desc: Perform IOs on multi-subsystems based NVMe-OF targets using fio tool with Multiple-Initiators
+      destroy-cluster: false
+      module: test_ceph_nvmeof_gateway.py
+      name: Test to run IOs using multiple-initiators against multi-subsystems based NVMe-of targets
```

Comment:
I think the name would be trimmed by the framework later, please keep name short.